### ISSUE TITLE
(0.20.5) system.src.js bad return value at line 3673

### ISSUE THREAD

**Issue by kudoz83** [systemjs/systemjs#1574]
I'm working through this, but looks like (at minimum) line 3673 in system.src.js....

```
 if (transpiler === metadata.pluginModule)
      return load.source;
```

should be 

```
 if (transpiler === metadata.pluginModule)
      return metadata.load.source;
```

Regardless of the change, I'm not seeing a value when using plugin-typescript. But `load `isn't a property in scope.

(plugin-typescript 6.0.4)
```
load === undefined. 
metadata.load === undefined. 
```

**Comment by guybedford** [comment]
Thanks for tracking this down, I've added a fix in https://github.com/systemjs/systemjs/commit/f1fbbfe015382577c6f633f72a1b853b204c893f.

This case applies when there is both a transpired and a plugin configured.

**Comment by guybedford** [comment]
Released in 0.20.7.

**Comment by kudoz83** [comment]
> This case applies when there is both a **transpired** and a plugin configured.

You mean _**transpiled**_! (its funny because there is another bug related to spelling... I'll be here all week.)
